app/templates/about.html

```diff
@@ -9,6 +9,28 @@ <h2>About the POLDER Federated Search App</h2>
     <p>Find out more by reading the <a class="about__link" href="https://polder-crew.github.io/Federated-Search-Documentation/intro.html" rel="noopener" target="_blank">documentation book</a> or exploring the <a class="about__link" href="https://github.com/nein09/polder-federated-search" rel="noopener" target="_blank">GitHub repository</a>.</p>
     <p>If you'd like to send us questions or comments directly, you are welcome to <a class="about__link" href="mailto:cverhey@oceannetworks.ca">email us</a>.</p>
 
+    
+        <div clas="covered_repositories">
```

Comment:
Whoa, looks like your 'about' changes crept in here, too! It'd be better to keep these on another branch so we can look at one thing at a time.

Reply:
I'll take a look at them now that they're in here, though, since there are a couple other changes that need to happen before I merge this anyway.